mycroft/tts/mimic_tts.py

```diff
@@ -32,26 +32,43 @@
 
 from .tts import TTS, TTSValidator
 
-CONFIG = Configuration.get().get("tts").get("mimic")
-DATA_DIR = expanduser(Configuration.get()['data_dir'])
 
-BIN = CONFIG.get("path",
-                 os.path.join(MYCROFT_ROOT_PATH, 'mimic', 'bin', 'mimic'))
+def get_mimic_binary():
+    """Find the mimic binary, either from config or from PATH.
 
-if not os.path.isfile(BIN):
-    # Search for mimic on the path
-    import distutils.spawn
+    Returns:
+        (str) path for mimic executable
+    """
+    config = Configuration.get().get("tts").get("mimic")
```

Comment:
since you are refactoring maybe clean up this line

if one happens to not have `"tts"` or `"mimic"` in the config this will throw an error in the next line on `config.get("path")` because `config is None`

should be replaced with something like `config = Configuration.get().get("tts", {}).get("mimic", {})`

this caused trouble and has been patched in [mycroft-lib](https://github.com/HelloChatterbox/mycroft-lib/commit/814223b2dd92beabd24717708c242008b1843641#diff-ac9c4c289f97ce804a71c1f3d63189d25078f3c895054004b936f5233070568bR35)

Reply:
Good suggestion. it never hurt to program defensively like that. will add that